src/sentry/conf/server.py

```diff
@@ -835,8 +835,6 @@ def create_partitioned_queues(name):
     "organizations:discover-query": True,
     # Enable Performance view
     "organizations:performance-view": False,
-    # Enable Performance in the Release View
-    "organizations:release-performance-views": False,
```

Comment:
I still see a few hits for this feature flag in the frontend code. Should those be cleaned up?

Reply:
Are you on latest master? They should be gone already.

Reply:
They're already removed in https://github.com/getsentry/sentry/pull/22259